static/js/analytics/agencies.js

```diff
@@ -36,11 +36,12 @@ $(document).ready(function () {
         },
         {
           render: function(data, type, row) {
-            if (type == "sort")
-              return null;
-            return Utils.progressBar(Utils.percent(
+            var percent = Utils.percent(
               row.analytics.participating, row.analytics.eligible
-            ));
+            )
+            if (type == "sort")
+              return percent;
+            return Utils.progressBar(percent);
```

Comment:
This fixes a bug reported to us by a user -- it was a regression from the refactor, where the rendered percent columns on the Agency pages for each category were not sorting correctly.